python/lsst/ts/standardscripts/maintel/setup_mtcs.py

```diff
@@ -0,0 +1,380 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["SetupMTCS"]
+
+import asyncio
+
+# import logging
+# import os
+# import sys
+import yaml
+
+import pandas as pd
+
+from lsst.ts import salobj, utils
+from lsst.ts.observatory.control.maintel.mtcs import MTCS
+
+
+class SetupMTCS(salobj.BaseScript):
+    """
+    Setup MTCS components so they are ready for operation.
```

Comment:
Move this to the line above, e.g.:

"""Setup MTCS ....

"""

Reply:
ping...

Reply:
Done